h/features/models.py

```diff
@@ -63,6 +80,10 @@ class Feature(Base):
                       default=False,
                       server_default=sa.sql.expression.false())
 
+    featurecohorts = sa.orm.relationship('FeatureCohort',
```

Comment:
This can probably just be `cohorts` -- because a `cohorts` attribute on a `feature` is self-evidently in reference to feature cohorts as opposed to any other kind of cohort.